packages/venia-ui/lib/components/Icon/icon.css

```diff
@@ -8,7 +7,7 @@
 
 .icon {
     fill: var(--fill, none);
-    stroke: var(--stroke);
+    stroke: var(--stroke, rgb(var(--venia-global-color-text)));
```

Comment:
Moving this here and falling back to this default value allows the parent to provide a stroke color w/o having to override the `classes` prop of Icon.